lib/webdriveragent.js

```diff
@@ -399,6 +403,14 @@ class WebDriverAgent {
   set url (_url) {
     this._url = url.parse(_url);
   }
+
+  get fullyStarted () {
```

Comment:
My main reason for having different names is that from within the `WebDriverAgent` class what "fully started" means at the moment is that we should log errors from iproxy. I'd like to keep it at that granularity, so that the actual conditions of its use are clear.

From outside, on the other hand, the calling code doesn't need to know that underneath we are suppressing logging. We just want to indicate that it is started.